nautobot/extras/forms.py

```diff
@@ -496,6 +497,7 @@ class Meta:
             "validation_minimum",
             "validation_maximum",
             "validation_regex",
+            "advanced_ui",
```

Comment:
I wonder if the UI would flow better if this field were earlier in the list, perhaps after "required"?